server/main.go

```diff
@@ -34,8 +36,14 @@ func main() {
 	check(err)
 
 	host, err := os.Hostname()
+
```

Comment:
Gah. I was having issues with the make file not remaking with just changes in cdn, only with changes in main. So add a space and save...a lot. Fixed.